searx/locales.py

```diff
@@ -188,7 +188,7 @@ def get_locale(locale_tag: str) -> Optional[babel.Locale]:
         return None
 
 
-def get_offical_locales(
+def get_official_locales(
```

Comment:
Be careful when changing function names .. but I have seen you replaced it in all references :+1: